### PR TITLE
Set up production deployment pipeline

### LINKED ISSUE #70: Figure out deployment setup for new framework
Can we do CI deploys?
What are the environment variables going to be?
- Auth0?
- Graphcool?

### PR DESCRIPTION
**Pull request by wdoug**
This PR closes #70 

#### What does this PR do?

Adds deployment of commits to the master branch to github pages (for the client) and graphcool's hosted backend (for the backend).

NOTE: Travis CI doesn't deploy Pull Requests, so this deploy pipeline was tested on a different branch for developing the process. See the pipeline output log [here](https://travis-ci.org/codefordenver/members/builds/323050534), and here is the resulting deployment: https://codefordenver.github.io/members

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,25 @@
 language: node_js
 node_js:
   - 8.4.0
-script: "npm run lint && npm test"
+
+script:
+  - "npm run lint"
```
Comment (dviramontes) [inline .travis.yml:6]: do these need to be in quotes ? / havent seen it like this b4

**Review by dviramontes — APPROVED** [review]
just the one question, the rest of looks great 👍 
thanks @wdoug !

File: .travis.yml (hunk 1 above)
Comment (wdoug) [reply .travis.yml:6]: No, it is optional for yaml syntax, but I think it clarifies things sometimes.

Comment (dviramontes) [reply .travis.yml:6]: gotcha 👍

**Pull request merged**